app/components/DraggableElement.js

```diff
@@ -3,8 +3,8 @@ import { DragSource } from 'react-dnd';
 import { getEmptyImage } from 'react-dnd-html5-backend';
 import shouldPureComponentUpdate from './utils/shouldPureComponentUpdate';
 import ItemTypes from './utils/ItemTypes';
-import TextElement from './TextElement';
-import PhotoElement from './PhotoElement'
+import whatTypeElementToRender from './utils/whatTypeElementToRender'
```

Comment:
Not a correction, just a question: Can we use this function for rendering videos in the suitcase?